roxctl/deployment/check/check.go

```diff
@@ -172,6 +174,12 @@ func (d *deploymentCheckCommand) Construct(_ []string, cmd *cobra.Command, f *pr
 }
 
 func (d *deploymentCheckCommand) Validate() error {
+	if d.cluster != "" && d.namespace == "default" {
+		d.env.Logger().WarnfLn("Warning: The namespace defaulted to \"default\", if you want to check deployments in a custom namespace set it with '--namespace [NAMESPACE]' or '-n [NAMESPACE]'\n")
```

Comment:
Same comment about `Warning` here.

Reply:
This message is a bit misleading. 

1. `default` (or `-n [namespace]`) namespace will only be used if the deployment yaml **doesn't** have a namespace set. 
2. "if you want to check deployments in a custom namespace" reads to me like we're checking the deployments that already exist in this namespace (this might just be my perception)

I suggest we rephrase to something like:
```
No custom namespace provided. Any deployments without an explicit namespace will be defaulted to "default". Overwrite the default with --namespace/-n [NAMESPACE]
```

Reply:
Done